build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -114,6 +114,19 @@ steps:
       vstsPackageVersion: '*'
     condition: and(succeeded(), eq(variables['VSCODE_QUALITY'], 'saw'))
 
+  - powershell: |
+      # Install SAW specific extensions
+      . build/azure-pipelines/win32/exec.ps1
+      $ErrorActionPreference = "Stop"
+      $tempFilePath = (New-TemporaryFile).FullName
+      $zipFilePath = $tempFilePath + ".zip"
+      $extensionUri = "$(standalone-extensions-uri)"
+      $adsExtensionPath = "$(agent.builddirectory)\azuredatastudio-win32-x64\resources\app\extensions"
+
+      exec { Invoke-WebRequest -Uri $extensionUri -OutFile $tempFilePath;Move-Item $tempFilePath $zipFilePath;Expand-Archive $zipFilePath -DestinationPath $adsExtensionPath; }
```

Comment:
Do we need the Move-Item part here? Could we use the zipFilePath for the OutFile argument of the web request?

Reply:
New-TempFile can't override extension. Always is .tmp. 

Expand-Archive can't deal with extensions other than .zip